src/toolchain/export-web.md

```diff
@@ -0,0 +1,157 @@
+It is likely to due a bug, but is still being investigated.  Edit the line to something like the following:
+
+```toml
+godot = { git = "https://github.com/godot-rust/gdext", branch = "master", features = ["experimental-wasm", "lazy-function-tables"] }
+```
+
+If you do not already have `.cargo/config.toml` file, do the following:
+
+- Create a `.cargo` directory at the same level as your `Cargo.toml`.
+- Inside of that directory, create a `config.toml` file.
+
+This file needs to contain the following:
+
+```toml
+[target.wasm32-unknown-emscripten]
+rustflags = [
+    "-C", "link-args=-sSIDE_MODULE=2",
+    "-C", "link-args=-pthread",
+    "-C", "target-feature=+atomics,+bulk-memory,+mutable-globals",
+    "-Zlink-native-libraries=no"
+]
+```
+
+Edit the project's `.gdextension` file to include support for web exports.
+This file will probably be at `godot/{YourCrate}.gdextension`.
+The format will be similar to the following:
+
+```ini
+[libraries]
+...
+web.debug.wasm32 = "res://../rust/target/wasm32-unknown-emscripten/debug/{YourCrate}.wasm"
+web.release.wasm32 = "res://../rust/target/wasm32-unknown-emscripten/release/{YourCrate}.wasm"
+```
+
+
+## Compile the Project
+
+Verify `emcc` is in the `PATH`. This can be as simple as doing the following:
+
+```sh
+emcc --version
+```
+
+Compile the code.
+It is necessary to both use the nightly compiler and specify to build std[^3], along with specifying the Emscripten target.
+
+```sh
+cargo +nightly build -Zbuild-std --target wasm32-unknown-emscripten
+```
+
+
+## Godot Editor Setup
+
+Add a web export in the Godot Editor. In the top menu bar, go to `Project > Export...` and configure it there.
+Make sure to turn on the `Extensions Support` checkbox.
+
+![Example of export screen](images/web-export.png)
+
+If instead the bottom on the export popup contains this error:
+> No export template found at expected path:
+
+![Example of missing template](images/web-missing-template.png)
+
+Then click on "Manage Export Templates" and then on the next screen click on "Download and Install"
+
+![Template download popup](images/web-download-template.png)
+
+Back at the main editor screen, there is an option to run the web debug build (_not_ a release build) locally
+without needing to run an export or set up a web server.
+At the top right, choose `Remote Debug > Run in Browser` and it will automatically open up a web browser.
+
+If your default browser is not Chromium-based (e.g., Firefox or Safari),
+you will need to copy the URL (which is usually `http://localhost:8060/tmp_js_export.html`)
+and open it in a supported browser such as Google Chrome or Microsoft Edge.[^4]
+
+![Location of built-in web server](images/web-browser-run.png)
+
+
+## Debugging
+
+Currently the only option for WASM debugging is
+[this extension](https://chromewebstore.google.com/detail/cc++-devtools-support-dwa/pdcpmagijalfljmkmjngeonclgbbannb?pli=1)
```

Comment:
Could you add the name of the extension in the link text, and use the `[<name of extension>][wasm-debugger]` format (i.e. move the actual link to the end of the section)?